charger/ocpp/connector.go

```diff
@@ -125,7 +127,9 @@ func (conn *Connector) Initialized() error {
 			return nil
 
 		case <-trigger: // try to trigger StatusNotification again as last resort
-			conn.TriggerMessageRequest(core.StatusNotificationFeatureName)
+			if conn.cp.HasRemoteTriggerFeature {
```

Comment:
This was intentionally not tested because there are boxes where the features are signaled incorrectly or not at all and RemoteTrigger still works.